src/main/java/org/broadinstitute/hellbender/utils/recalibration/RecalibrationArgumentCollection.java

```diff
@@ -108,6 +108,12 @@
     @Argument(fullName = "computeIndelBQSRTables", shortName = "indelBQSR", doc = "compute indel BQSR tables")
     public boolean computeIndelBQSRTables = false;
 
+    @Argument(fullName="readShardSize", shortName="readShardSize", doc = "Maximum size of each read shard, in bases.", optional = true)
+    public int readShardSize = 10000;
+
+    @Argument(fullName="readShardPadding", shortName="readShardPadding", doc = "Each read shard has this many bases of extra context on each side.", optional = true)
+    public int readShardPadding = 1000;
+
```

Comment:
These should not be in `RecalibrationArgumentCollection`, since they are specific to Spark (and to one particular Spark sub-implementation). Can you either move to `BaseRecalibratorSpark`/`BQSRPipelineSpark` or, if you'd prefer to avoid repetition, to your own `OverlapsPartitionerArgumentCollection`/`SparkSharderArgumentCollection` included by BQSR and its various pipelines? 

Also adjust the argument docs to make it clear that they only apply when using the `OVERLAPS_PARTITIONER` join strategy.